src/server/dom.ts

```diff
@@ -884,7 +884,7 @@ export function throwFatalDOMError<T>(result: T | FatalDOMError): T {
   if (result === 'error:notinput')
     throw new Error('Node is not an HTMLInputElement');
   if (result === 'error:notinputvalue')
```

Comment:
Now that it isn't input, we can rename `error:notinputvalue` to `error:hasnovalue`.

Reply:
Done